tests/integration/test_data_rows.py

```diff
@@ -685,12 +685,24 @@ def test_create_data_rows_sync_mixed_upload(dataset, image_url):
 
 def test_delete_data_row_attachment(datarow, image_url):
     attachments = []
+
+    # Anonymous attachment
     to_attach = [("IMAGE", image_url), ("TEXT", "test-text"),
                  ("IMAGE_OVERLAY", image_url), ("HTML", image_url)]
     for attachment_type, attachment_value in to_attach:
         attachments.append(
             datarow.create_attachment(attachment_type, attachment_value))
 
+    # Attachment with a name
+    to_attach = [("IMAGE", image_url, "Att. Image"),
```

Comment:
You probably want to add to `test_data_row_attachments` and `test_create_data_rows_sync_attachments ` as well to assert that the name gets populated when specified

Reply:
Thanks, @cssetian. Following your message, I updated the two other tests.